src/glwidget.cpp

```diff
@@ -293,38 +296,39 @@ void GLWidget::setZRotation(int angle)
     qNormalizeAngle(angle);
     zRot = angle;
     is2DView = false;
-    updateGL();
+    update();
     return;
 }
 
 void GLWidget::setXDist(int angle)
 {
     xDist = angle;
     is2DView = false;
-    updateGL();
+    update();
     return;
 }
 
 void GLWidget::setYDist(int angle)
 {
     yDist = angle;
     is2DView = false;
-    updateGL();
+    update();
     return;
 }
 
 void GLWidget::setZDist(int angle)
 {
     zDist = angle;
     is2DView = false;
-    updateGL();
+    update();
     return;
 }
 
 
 void GLWidget::initializeGL()
 {
-    qglClearColor(Qt::black);
+    QColor color = Qt::black;
```

Comment:
Trivial performance improvement ([docs](https://doc.qt.io/qt-5/qcolorconstants.html#details)):

```suggestion
#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
    QColor color(QColorConstants::Black);
#else
    QColor color(Qt::black);
#endif
```